src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1766,8 +1766,8 @@ def __init__(self,
                  placement: Optional[PlacementSpec] = None,
                  disable_https: Optional[bool] = False,
                  port: Optional[int] = None,
-                 ssl_certificate: Optional[str] = None,
-                 ssl_certificate_key: Optional[str] = None,
+                 ssl_certificate: Optional[List[str]] = None,
```

Comment:
do we really want these to be lists of strings rather than strings? How are we expecting them to be passed? This format
```
client_cert: |
    -----BEGIN CERTIFICATE-----
    ...
    -----END CERTIFICATE-----
```
for example would just be a string and that's the one we tend to document nowadays

Reply:
I changed the field to be only string and removed all the references/logic to strings list